### PR TITLE
838658 - Added the busy indicator animation types gif to the WPF UG documentation

### PR DESCRIPTION
**Pull request by Arunachalam3641**
Added the busy indicator animation types gif to the WPF UG documentation

### Output Image before implementation 
image

### Output Image After implementation 
image

### CODE REVIEW

**Review by Choza-rajan — APPROVED** [review]

File: wpf/Busy-Indicator/AnimationTypes.md [hunk 1]
```diff
@@ -36,201 +36,8 @@ busyIndicator.AnimationType = AnimationTypes.Flight;
 {% endtabs %}
 
 
-![ArrowTrack](AnimationTypes_images/AnimationTypes_img1.jpeg)
-
-Busy Indicator with ArrowTrack type animation
-{:.caption}
-
-
-![Ball](AnimationTypes_images/AnimationTypes_img2.jpeg)
```
Comment (Choza-rajan) [inline wpf/Busy-Indicator/AnimationTypes.md:45]: Delete all unwanted animation images

Comment (Arunachalam3641) [reply wpf/Busy-Indicator/AnimationTypes.md:45]: Yes, I deleted all unwanted animation images

**Comment by Arunachalam3641** [comment]
rebuid

**Comment by Choza-rajan** [comment]
The changes are fine

**Pull request merged**